api/routers/edr.py

```diff
@@ -112,13 +114,23 @@ async def get_locations(
     # Check for multiple coordinates or names on one station
     for station_id in platform_parameters.keys():
         if len(platform_coordinates[station_id]) > 1:
-            raise HTTPException(
-                status_code=500,
-                detail={
-                    "coordinates": f"Station with id `{station_id} "
-                    f"has multiple coordinates: {platform_coordinates[station_id]}"
-                },
-            )
+            if (
+                calculate_largest_postition_deviation(platform_coordinates[station_id]) < 1e-4
+            ):  # all coordinates are within 1e-4 degrees (roughly 10 m)
+                platform_coordinates[station_id] = {
+                    sorted(
+                        [i for i in platform_coordinates[station_id]],
+                        key=lambda x: (len(str(x[0])), len(str(x[1])), x[0], x[1]),
+                    )[-1]
+                }
+            else:
+                raise HTTPException(
+                    status_code=500,
+                    detail={
+                        "coordinates": f"Station with id `{station_id} "
+                        f"has multiple coordinates: {platform_coordinates[station_id]}"
```

Comment:
Write something like "multiple incompatible coordinates", so it is clear what we are actually testing.